include/ibm/locks.hpp

```diff
@@ -261,7 +261,7 @@ inline RcAcquireLock Lock::acquireLock(const LockRequests& lockRequestStructure)
 
     if (status)
     {
-        BMCWEB_LOG_DEBUG << "There is a conflict within itself";
+        BMCWEB_LOG_ERROR << "There is a conflict within itself";
```

Comment:
We really want an ERROR here?

Reply:
> We really want an ERROR here?

This is to get traces in the journal log if there is a lock conflict issue. There is no other way to identify this otherwise